socorro/external/es/crashstorage.py

```diff
@@ -17,6 +18,10 @@
 from socorro.external.crashstorage_base import Redactor
 
 
+FIELD_NAME_STRING_ERROR_REGEX = re.compile(r'field=\"([\w.]+)\"')
+FIELD_NAME_NUMBER_ERROR_REGEX = re.compile(r'\[failed to parse \[([\w.]+)]]')
```

Comment:
Nit: We might want to move these INTO the class `ESCrashStorage` as class attributes. 
There's no need to put them on a module level since we're not importing them from other places.